manifests/native.json

```diff
@@ -1047,6 +1047,22 @@
       "replacement": "Object.assign, or if deep clones are needed, use structuredClone",
       "mdnPath": "Global_Objects/Object/assign",
       "category": "native"
+    },
+    {
```

Comment:
I think we should only have it in the preferred manifest. Otherwise we might cause duplicate warnings in linters etc

Reply:
Hmm ok. But then, I guess other modules will have similar issues, does it even make sense to maintain a native manifest?

Reply:
so far, any given module has only been in one manifest

the native one makes sense for those modules which have an exact native equivalent. e.g. if we only suggested using `Intl` and not a third party package, it'd probably exist only in the native manifest

mentioning the third party package means it belongs in the preferred manifest with documentation

I think that's fine though, its good to mention the package too